src/client/MetaRequest.h

```diff
@@ -47,12 +53,12 @@ struct MetaRequest {
   __u32    sent_on_mseq;       // mseq at last submission of this request
   int      num_fwd;            // # of times i've been forwarded
   int      retry_attempt;
-  std::atomic<uint64_t> ref = { 1 };
-  
+  std::atomic<uint64_t> ref = { 0 };
```

Comment:
When building ref counted objects, it's better to inherit from `RefCountedObject` in `src/common`.

Reply:
Sure, I will do that.

Reply:
Done.